Initial layout of new doc structure (#284)
All-Hands-AI/docs: openhands/usage/essential-guidelines/sdlc-integration.mdx

---
title: TODO-OpenHands in Your SDLC
description: How OpenHands fits into your software development lifecycle
---

> OpenHands can enhance every phase of your software development lifecycle (SDLC), from planning through deployment. This guide shows how to integrate OpenHands into your development workflows effectively.

Reviewer: ```suggestion
OpenHands can enhance every phase of your software development lifecycle (SDLC), from planning through deployment. This guide shows some example prompts that you can use when you integrate OpenHands into your development workflow.
```
